packages/admin-ui/src/filters/filter-status.tsx

```diff
@@ -17,12 +19,17 @@ export function FilterStatus(props: FilterStatusProps) {
   if (!status) return null
 
   // more filter status comming soon
-  if (status === 'ready' || status === 'loading' || status === 'empty')
+  if (
+    status === 'ready' ||
+    status === 'loading' ||
+    status === 'empty' ||
+    status === 'error'
+  )
```

Comment:
Suggestion:
```suggestion
 if (["ready", "loading", "empty", "error"].includes(status)) {
  return null
}

```

Reply:
Sounds ok, but why?

Reply:
It is more a suggestion, but it is just a first possible change, because we can see some status being checking more than once... I was just wondering if it could make sense let those checking more explicit...as creating a function which represents those checking...